src/sphinx/code/SqlToSlick.scala

```diff
@@ -43,7 +43,7 @@ object SqlToSlick extends App {
   val db = Database.forConfig("h2mem1")
   try {
 
-    Await.result(db.run(Action.seq(
+    Await.result(db.run(DBIO.seq(
```

Comment:
no wait. What is the difference between DBIO.seq and DBIO.sequence? can we rename .seq to something else?